extensions/wikia/UserProfilePageV3/UserIdentityBoxDiscussionInfo.class.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+use Swagger\Client\ApiException;
+use Swagger\Client\Discussion\Api\ContributionApi;
+use Swagger\Client\Discussion\Api\SitesApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+
+class UserIdentityBoxDiscussion {
+
+	const DISCUSSION_SERVICE_NAME = 'discussion';
+	const TIMEOUT = 5;
+
+	private $userId = null;
+	private $discussionActive = false;
+	private $discussionPostsCount = 0;
+	private $discussionAllPostsByUserLink = '';
+
+	function __construct($user) {
+		$this->userId = $user->getId();
+	}
+
+	public function isDiscussionActive() {
+		return $this->discussionActive;
+	}
+
+	public function getDiscussionPostsCount() {
+		return $this->discussionPostsCount;
+	}
+
+	public function getDiscussionAllPostsByUserLink() {
+		return $this->discussionAllPostsByUserLink;
+	}
+
+	public function fetchDiscussionPostsNumber() {
+		wfProfileIn( __METHOD__ );
```

Comment:
These wfProfileIn|Out calls are unnecessary.  They come from a time when there was no other way in PHP to profile call times.  Now xdebug supports this and we send this information to influxdb from within the PHP interpreter rather than within our code.